packages/react/src/components/Authenticator/SetupTOTP/__tests__/SetupTOTP.test.tsx

```diff
@@ -88,4 +88,28 @@ describe('SetupTOTP', () => {
     expect(toDataURLSpy).toHaveBeenCalledTimes(1);
     expect(toDataURLSpy).toHaveBeenCalledWith(customTotpCode);
   });
+
+  it('handles customTotpIssuer with spaces', async () => {
```

Comment:
Nit: could just place this test its own `describe` block and remove the component code, might make it easier for future readers:
```
describe('getTotpCode', () => {
  it('handles customTotpIssuer with spaces', () => {
    const customTotpIssuer = 'customTOTPIssuer spaces';
    const customTotpUsername = 'customTotpUsername';

    const customTotpCode = getTotpCode(
      customTotpIssuer,
      customTotpUsername,
      SECRET_KEY
    );

    expect(customTotpCode).toBe(
      'otpauth://totp/customTOTPIssuer%20spaces:customTotpUsername?secret=secretKey&issuer=customTOTPIssuer%20spaces'
    );
  });
});
```

Reply:
OK I'll move it out into a new describe